### PR TITLE
add template and scaffold for vue and magic guide

### PR DESCRIPTION
**Pull request by dgerrellsMagic**
### 📦 Pull Request

Add template and scaffold for vue + magic guide. 

### ✅ Fixed Issues

NA

### 🚨 Test instructions

NA

### CODE REVIEW

File: scaffolds/hello-world-vue/scaffold.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <Template source="./src" />
+      <Template source="./.env" />
+      <Template source="./.gitignore" />
+      <Template source="./.eslintrc.js" />
+      <Template source="./babel.config.js" />
+      <Template source="./package.json" />
+      <Template source="./README.md" />
+      <Zombi {...props} data={false}>
+        <Template source="./public" />
+      </Zombi>
```
Comment (smithki) [inline scaffolds/hello-world-vue/scaffold.tsx:21]: We can actually skip the nested `<Zombi>` here, the `<Template />` component also accepts a `data` prop. So we can do this:

```tsx
<Template source="./public" data={false} />
```

**Review by smithki — APPROVED** [review]
Just one comment. Fix n' sheep! 🐑

File: scaffolds/hello-world-vue/scaffold.tsx (hunk 1 above)
Comment (smithki) [reply scaffolds/hello-world-vue/scaffold.tsx:21]: Also, just for your knowledge: the order of `data` precedence is `<Template>` first, then `<Zombi>` as a fallback.

Comment (dgerrellsMagic) [reply scaffolds/hello-world-vue/scaffold.tsx]: Fixed.

**Pull request merged**